trestle/core/commands/split.py

```diff
@@ -37,14 +37,16 @@
 
 def split_is_too_fine(split_paths: str, model_obj: OscalBaseModel) -> bool:
```

Comment:
Ensure an issue exists to track fixing this.

Reply:
done